### PR TITLE
Flake-ctl-build-debbian

### PR DESCRIPTION
**Pull request by Ichmed**

### CODE REVIEW

**Review by isbm — CHANGES_REQUESTED** [review]
@Ichmed could you please look at these ~disasters~ discrepancies? :-)

File: flake-ctl/flake-ctl-build-dpkg/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+use flakes::config::itf::FlakeConfig;
+
+fn main() -> Result<()> {
+    flake_ctl_build::run::<DPKGBuilder>()
+}
+
+#[derive(Debug, Args)]
+struct DPKGBuilder {
+    /// Location of .spec template and pilot specific data
+    #[arg(long, default_value = "/usr/share/flakes/package/dpkg")]
```
[the remark this answers is not in the document]
Comment (Ichmed) [reply flake-ctl/flake-ctl-build-dpkg/src/main.rs:19]: also no spec template here

**Review by isbm — CHANGES_REQUESTED** [review]
I am not entirely convinced of mere existence of [this whole thing](https://github.com/Elektrobit/flake-pilot/blob/85d054d48caa1fcfa8d9b150f6c16eb24dfcf2e3/flake-ctl/flake-ctl-firecracker/src/defaults.rs#L24-L41). I still believe that this should be using (or extending if something is still missing) the central config instead.

**Comment by Ichmed** [comment]
> I am not entirely convinced of mere existence of [this whole thing](https://github.com/Elektrobit/flake-pilot/blob/85d054d48caa1fcfa8d9b150f6c16eb24dfcf2e3/flake-ctl/flake-ctl-firecracker/src/defaults.rs#L24-L41). I still believe that this should be using (or extending if something is still missing) the central config instead.

Is that referenced somewhere? otherwise i agree but that should not be done in this PR

**Review by isbm — APPROVED** [review]
@Ichmed OK, fine. Let's incorporate `defaults.rs` outdated stuff with the new config in the next PR. Good to go here.

**Pull request merged**